cfgov/unprocessed/root/robots.txt

```diff
@@ -13,5 +13,6 @@ Disallow: /ask-cfpb/search-by-tag/
 Disallow: /askcfpb/search/
 Disallow: /es/obtener-respuestas/buscar/
 Disallow: /es/obtener-respuestas/buscar-por-etiqueta/
+Disallow: /consumer-tools/educator-tools/youth-financial-education/curriculum-review/tool/*
```

Comment:
Not sure if it matters, but this will also disallow crawling of the top-level tool page, `/consumer-tools/educator-tools/youth-financial-education/curriculum-review/tool/`, which looks like it redirects to `/consumer-tools/educator-tools/youth-financial-education/curriculum-review/before-you-begin/` if you haven't started a review yet. I *think* `/consumer-tools/educator-tools/youth-financial-education/curriculum-review/before-you-begin/` will still get crawled even if we disallow a redirect to it, so it might not matter.

Reply:
I was intending to disallow `/tool/` and anything that follows, including hash strings and query strings, to exclude any URL that exposes data entry.